lib/webpack/railtie.rb

```diff
@@ -30,7 +30,9 @@ class Railtie < ::Rails::Railtie
     config.webpack.dev_server.manifest_host = 'localhost'
     config.webpack.dev_server.manifest_port = 3808
 
-    config.webpack.dev_server.https = false # note - this will use OpenSSL::SSL::VERIFY_NONE
+    config.webpack.dev_server.https = false
+    # change below option to OpenSSL::SSL::VERIFY_NONE if you don't care about security
+    config.webpack.dev_server.https_mode = 'OpenSSL::SSL::VERIFY_PEER'
```

Comment:
Change to `config.webpack.dev_server.https_verify_peer`, default to true. If true, use VERIFY_PEER otherwise VERIFY_NONE.